ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/reset/ResetServiceIT.scala

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.services.reset
+
+import java.io.File
+import java.util.UUID
+
+import com.digitalasset.daml.bazeltools.BazelRunfiles.rlocation
+import com.digitalasset.ledger.api.domain.LedgerId
+import com.digitalasset.ledger.api.testing.utils.{
+  IsStatusException,
+  SuiteResourceManagementAroundEach,
+  MockMessages => M
+}
+import com.digitalasset.ledger.api.v1.active_contracts_service.{
+  ActiveContractsServiceGrpc,
+  GetActiveContractsRequest,
+  GetActiveContractsResponse
+}
+import com.digitalasset.ledger.api.v1.command_completion_service.{
+  CommandCompletionServiceGrpc,
+  CompletionStreamRequest
+}
+import com.digitalasset.ledger.api.v1.command_service.{CommandServiceGrpc, SubmitAndWaitRequest}
+import com.digitalasset.ledger.api.v1.command_submission_service.CommandSubmissionServiceGrpc
+import com.digitalasset.ledger.api.v1.event.CreatedEvent
+import com.digitalasset.ledger.api.v1.ledger_identity_service.{
+  GetLedgerIdentityRequest,
+  LedgerIdentityServiceGrpc
+}
+import com.digitalasset.ledger.api.v1.testing.reset_service.{ResetRequest, ResetServiceGrpc}
+import com.digitalasset.ledger.api.v1.transaction_filter.TransactionFilter
+import com.digitalasset.platform.common.LedgerIdMode
+import com.digitalasset.platform.sandbox.config.SandboxConfig
+import com.digitalasset.platform.sandbox.services.{SandboxFixture, TestCommands}
+import com.digitalasset.platform.testing.{FiniteStreamObserver, WaitForCompletionsObserver}
+import com.digitalasset.timer.RetryStrategy
+import com.google.protobuf.empty.Empty
+import io.grpc.Status
+import org.scalatest.concurrent.{AsyncTimeLimitedTests, ScalaFutures}
+import org.scalatest.time.Span
+import org.scalatest.{AsyncWordSpec, Matchers}
+
+import scala.concurrent.Future
+import scala.concurrent.duration.{Duration, DurationInt, DurationLong}
+
+final class ResetServiceIT
+    extends AsyncWordSpec
+    with AsyncTimeLimitedTests
+    with Matchers
+    with ScalaFutures
+    with SandboxFixture
+    with SuiteResourceManagementAroundEach
+    with TestCommands {
+
+  private val eventually = RetryStrategy.exponentialBackoff(10, 10.millis)
```

Comment:
Why not use ScalaTest's `eventually`?

Reply:
I'm not sure of the reason why but everything fails if I use that. I'm not sure of the semantics there.